### PR TITLE
Update webclient documentation

### PR DESCRIPTION
**Pull request by klustria**

### CODE REVIEW

**Comment by klustria** [comment]
This PR was created for issue https://github.com/helidon-io/helidon/issues/6456

File: docs/se/webclient.adoc [hunk 1]
```diff
@@ -157,94 +303,87 @@ client:
...
+==== Configuring Proxy in the config file
 
-JSON Processing (JSON-P) media support is not present in the WebClient by default. So, in this case, you must first register it before making a request.
-This example shows how to register `JsonpSupport` using the following two methods.
+Proxy can also be configured in WebClient through the `application.yaml` configuration file.
 
-[source,java]
-.Register JSON-P support to the WebClient.
+[source,yaml]
+.WebClient TLS configuration file `application.yaml`
```
Comment (Verdent) [inline docs/se/webclient.adoc:370]: I think you overlooked here. It should be Proxy instead of TLS

**Review by Verdent — DISMISSED** [review]
One very minor thing, but otherwise it looks good

File: docs/se/webclient.adoc (hunk 1 above)
Comment (klustria) [reply docs/se/webclient.adoc]: Corrected and did the same on the `Service` section

**Review by ljamen — APPROVED** [review]
LGTM - I'll make any small changes after the merge.

File: docs/se/webclient.adoc (hunk 1 above)
Comment (ljamen) [reply docs/se/webclient.adoc]: resolved

**Pull request merged**